### ISSUE TITLE
Switching back to php@7.4 always says it's not installed and has to install again via Brew

### ISSUE THREAD

**Issue by sebszocinski** [laravel/valet#1001]
As the title suggests, every time I switch from php 7.4 to another version and then back to 7.4 it always says:
```
Installing php@7.4...
[php@7.4] is not installed, installing it now via Brew... 🍻
```
Is there a way to keep 7.4 installed?

on MacOS Catalina and Laravel Valet 2.13.0

**Comment by drbyte** [comment]
It's unlikely that it's actually re-installing it.
Chances are the "install" takes mere seconds while it realizes in the background that there's no action required.

Here's why:
Homebrew treats `php` as the latest version. 
And in the background aliases the `php@7.4` (or whatever is newest at the time) to that.

Then later when you tell it (directly or via Valet) to install `php@7.4` it finds that `php@7.4` as a formula isn't installed, so begins trying to install it. Then in background it realizes it's already there due to internal aliases, and completes the "install" very fast by just failing out. Valet hides that "failure" silently.

So, in short, it's harmless.

But if it really annoys you, you can manually `brew remove php` and then `brew install php@7.4` ... and then `valet use php@7.4` to reconfigure it appropriately for valet.

**Comment by sebszocinski** [comment]
Sorry I should have elaborated further, it does seem to be re-installing it as it takes much longer to switch compared to when I downgrade to 7.3 or 7.2 - those take seconds.

**Comment by drbyte** [comment]
Then do the remove/install I mentioned.

**Comment by sebszocinski** [comment]
ok great thanks
